helm/oauth2-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 5.0.6
```

Comment:
Is this a breaking change?

Reply:
This is not a breaking change hence the minor version being bumped. This is only adding functionality in a backwards compatible manner.